lib/diplomat/datacenter.rb

```diff
@@ -6,8 +6,9 @@ class Datacenter < Diplomat::RestClient
     # Get an array of all avaliable datacenters accessible by the local consul agent
     # @param meta [Hash] output structure containing header information about the request (index)
     # @return [OpenStruct] all datacenters avaliable to this consul agent
-    def get(meta = nil)
+    def get(meta = nil, options = nil)
       url = ['/v1/catalog/datacenters']
+      url << use_consistency(options) if use_consistency(options, nil)
```

Comment:
why use 'if use_consistency' everywhere while it could be integrated into use_consistency (or a wrapper of both...) it's not very conservative if you forget one use_consistency call

Reply:
Did it in next patch